src/routes/v1/challenge.route.js

```diff
@@ -0,0 +1,13 @@
+const express = require("express");
+const router = express.Router();
+const challengeController = require("../../controllers/challenge.controller");
+
+router
+  .route("/")
+  .get((req, res) => challengeController.getRandomChallenge(req, res));
+
+router
+  .route("/:id")
+  .get((req, res) => challengeController.getChallengeById(req, res));
+
+module.exports = router;
```

Comment:
Swagger?

Reply:
I was going to do it in a different PR.  But I can do it on this one